src/components/features/ClientFeedback/ClientFeedback.js

```diff
@@ -34,6 +53,21 @@ class ClientFeedback extends React.Component {
       );
     }
 
+    const swipeContent = [];
+    for (let page = 0; page < pagesCount; page++) {
+      swipeContent.push(
+        <div className='row'>
```

Comment:
Do `<div className='row'>` pushujesz znowu `<div className='row'>`. Taka jest formuła przy swiperze, że rodzic i dziecko jest taki sam czy to błąd? Pytam z ciekawości.

Reply:
@MaciekLubocki możesz się proszę odnieść do uwagi @akijania

Reply:
Aniu, dziękuję dopiero teraz zauważyłem Twoje pytanie. Miałaś rację, niepotrzebny był <div className='row'> w każdym kolejnym pushu. Poprawiłem.

Reply:
Pisałam swojego taska wzorując się na Twoich plikach. Nie wiedziałam jak mam zrobić stąd to pytanie.

Reply:
Heh nie miałem pojęcia Ania że ktoś może na podstawie moich wypocin się wzorować, tym bardziej, że mocno kuleję i robię błędy. Także jeszcze raz dziękuję za cenną uwagę.

Reply:
Wszyscy robimy jeszcze masę błędów. Dla mnie wprowadzenie przewijania w moim tasku było bardzo trudne i nie wiem czy wszystkiego nie pokręciłam.